src/Microsoft.Diagnostics.Monitoring.WebApi/Stacks/CallStackData.cs

```diff
@@ -11,6 +11,13 @@ namespace Microsoft.Diagnostics.Monitoring.WebApi.Stacks
     /// </summary>
     internal sealed class CallStackResult
     {
+        public CallStackResult(NameCache nameCache)
+        {
+            NameCache = nameCache;
+        }
+
+        public CallStackResult() { }
```

Comment:
```suggestion
        public CallStackResult() : this(new NameCache()) { }
```

and remove the property value initializer for `NameCache`.

Reply:
I ended up just reverting this, since we no longer need the constructor if we take https://github.com/dotnet/dotnet-monitor/pull/4471#discussion_r1228775857